BatteryFactor.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BatteryManagement
+{
+    public class BatteryFactor
+    {
+
+        BatteryStatusDisplay BatteryStatusDisplay = new BatteryStatusDisplay();
+        public bool CheckBatteryCondition(string BatteryFactor, float minBatteryValue, float maxBatteryValue, float BatteryValue)
+        {
+            bool BatteryStatus;
+            if (BatteryValue < minBatteryValue)
+            {
+                BatteryStatus = BatteryStatusDisplay.PrintMinimumLimit(BatteryFactor, minBatteryValue, BatteryValue);

```

Comment:
argument names must be written using camel convention

Reply:
Refactored the code